index.html

```diff
@@ -3862,6 +3855,15 @@ <h4>updated</h4>
                 </p>
               </section>
 
+          <section>
+            <h4>deactivated</h4>
+            <p>
+This property MUST be populated with a <a data-cite="INFRA#boolean">boolean</a> 
```

Comment:
```suggestion
This property is populated with a <a data-cite="INFRA#boolean">boolean</a> 
```

Reply:
I prefer to keep this normative instead of declarative.

Reply:
If we keep this normative, how do we test this without wiring up a DID Method, creating a DID, deactivating it, and then querying the DID Method through the resolver interface to check this property? In other words, that sounds like a lot of effort for the WG to check a feature that we are not expecting people to mis-implement (and which should probably be in the DID Resolution test suite), no?

Reply:
The same way we test numbers and booleans today :)

https://did.actor/mike/did.json

Reply:
https://github.com/w3c/did-test-suite/pull/20/files#diff-7828e42130aa848d05937900dd5476d3978c431142ee878fe4388b87b92d129fR18

Reply:
It's not the "check to make sure it's a boolean" part of the normative state that I'm concerned about. It's the "if the DID Method determines that the DID supplied has been deactivated." part of the normative statement that's an issue. The test would require you to query a live system where a DID isn't deactivated... then deactivate it... then test to see if the value you get back would be set to "deactivated===true".

Reply:
The entire class of tests associated with sequential did operations should be considered out of scope... instead we should provide examples of resolution results for after: "created", "updated", "deactivated"... this is how we establish conformance in sidetree across languages and implementations... even @csuwildcat has agreed to those tests :)

Reply:
>  instead we should provide examples of resolution results for after: "created", "updated", "deactivated"...

Then the normative language would need to change to not include the decision process of a DID Method. We could reword this statement to:

> This property MUST be populated by a boolean value that is expected to be `true` if the DID has been deactivated, and `false` if the DID is still active.

... but that's just saying, the value MUST be a boolean value. It allows us to write two tests -- one where it's true and another where it's false. I question the value of those tests, but if folks really want to have normative language around the range of values and when they're expected to be those values, the language above would do that.

Reply:
The WG proposed the following text in a special topic call, there were no objections: This property MUST be populated by a boolean value that is true if the DID has been deactivated, and false if the DID is still active.

I'm going to hold the merge until we hear from @csuwildcat to make sure he's ok w/ the language.

@csuwildcat, are you ok with the proposed language above?

Reply:
I agree with this language. Unless @csuwildcat has any objections, we should accept this suggestion and then merge the PR.